src/components/varsler/VarslerNy.tsx

```diff
@@ -22,30 +59,87 @@ const DittNavInformasjonsLinje = ({
 }) => {
     return (
         <HStack gap="4">
-            <div className="text-sm font-bold">{tittel}</div>
+            <div className="font-bold">{tittel}</div>
             <div className={`${className} text-base`}>{tekst}</div>
         </HStack>
     );
 };
 
-const DittNavInformasjonsLinjer = (varsel: {
-    produsent: string;
-    tekst: string;
-    link: string;
+const DittNavInformasjonsLinjer = ({
+    varsel,
+    kanaler
+}: {
+    varsel: Varsel;
+    kanaler: string[];
 }) => {
     return (
         <VStack gap="1" className="p-2">
+            <HStack justify="space-between">
+                <Heading level="1" size="medium">
+                    {varsel.tekst}
+                </Heading>
+                <Link href={varsel.link}>
+                    <LinkIcon fontSize="1.5rem" />
+                </Link>
```

Comment:
Hvordan blir dette med accessabillity? Jeg tipper vi må ha noe tekst her og gjemme ikonet for skjermlesere.